src/components/visualization/layout/Visualization.tsx

```diff
@@ -22,7 +22,7 @@ export default function Visualization({ resource }: { resource: Resource }) {
     return <ChartTableWrapper resource={resource} transformedData={transformedData} />;
   } else if (resource.type === 'GeoJSON') {
     const geoJsonData = data as GeoJSON.FeatureCollection;
-    const Map = dynamic(() => import('@/components/visualization/Map'), {
+    const Map = dynamic(() => import('@/components/visualization/map/GeoJSONMap'), {
```

Comment:
If the component is now called `GeoJSONMap` the `const` should have the same name. (Although I did prefer the simpler name `Map`, personally)

Reply:
Ah, good point. Yes, I also preferred Map, this unfortunately led to problems with using a regular Map inside of GeoMap for collecting the labels.